src/main/java/stsjorbsmod/JorbsMod.java

```diff
@@ -418,4 +419,13 @@ public void receivePowersModified() {
             }
         }
     }
+
+    @Override
+    public void receiveStartAct() {
+        for (AbstractCard c : AbstractDungeon.player.masterDeck.group) {
+            if (c instanceof StartActSubscriber) {
+                ((StartActSubscriber)c).receiveStartAct();
```

Comment:
Repeating a comment from #476 that also applies here:

> I think forwarding the `StartGameSubscriber` and `StartActSubscriber` notifications is a perfectly reasonable design for Equinox. However, I would not recommend forwarding this notification to *every* master deck card that implements  the BaseMod subscribers like this, because it would be possible in theory for other mods to exist that have cards of their own which implement the BaseMod subscribers and have their own separate mechanisms for forwarding the same events, and we don't want to cause such a mod to receive 2 copies of those events.
>
> I think having `JorbsMod` implement BaseMod's `receiveStartAct` and `receiveStartGame` is good, but I'd suggest not using the BaseMod interfaces for the card; instead, I'd recommend adding equivalent methods on `CustomJorbsModCard` (with no-op default behavior), and only have the `JorbsMod` hooks invoke the `CustomJorbsModCard` hooks.

Reply:
Actually, I just remembered that this particular start-of-act behavior also needs to apply to base game cards that get exerted, so this probably should just inline the "reset exerted" behavior

Reply:
Thanks for the review! There's a couple things I don't quite understand regarding the implementation of your requests. Maybe there's just something I'm missing from how hooks can be implemented.
- Since we can set base cards to be Exerted, we still want to call the receiveStartAct method for each card in the MasterDeck, right? And not just for instanceof CustomJorbsModCard? But if I check card instanceof StartActSubscriber, the base cards won't receive this event, do they?
- Where do I place the logic for setting ExertedField to false at receiveStartAct? This should be general game logic, so putting it in a CustomJorbsModCard method would not work right for base cards? 

Hope you don't mind providing a little more guidance on this one!

Reply:
Ah, for some reason I didn't see your last reply. I'll push my current version. It does the reset of Exerted in Jorbsmod::receiveStartAct for all cards in MasterDeck. It also calls receiveStartAct for each card that's an instanceof StartActSubscriber. And lastly I've added your suggested pattern in CustomJorbsModCard. Hope I'm doing all that stuff correctly this time!

Reply:
The exert reset looks good in the current iteration, but this part:

> It also calls receiveStartAct for each card that's an instanceof StartActSubscriber.

...is the thing that still concerns me.

Consider what happens if *a second mod* that the user had installed were to try to implement similar functionality in the same way; two different mods' entry point classes would *both* be invoking the `receiveStartAct` hook on each card in the master deck that implements BaseMod's `StartActSubscriber`, so each such card would end up running its hooks multiple times.

We can avoid such an issue by just avoiding the use of the BaseMod `StartActSubscriber` interface on our cards or our `CustomJorbsModCard` base class; you can still have a method by the same name on the `CustomJorbsModCard` base class, just don't implement the BaseMod interface anywhere except `JorbsMod`, and have `JorbsMod`'s `receiveStartAct` hook look for cards that implement `CustomJorbsModCard` rather than cards that implement BaseMod's `StartActSubscriber`.

Reply:
Oh, of course! I was still thinking we needed send this event to all cards to get the Exert reset stuff to work, but of course that's not the case anymore. 
Just pushed a correction. Hopefully that's the last iteration.